sky/skylet/job_lib.py

```diff
@@ -377,6 +454,18 @@ def _get_jobs_by_ids(job_ids: List[int]) -> List[Dict[str, Any]]:
     return records
 
 
+def _get_pending_jobs():
```

Comment:
In L232 and L233, both `PENDING` and `RUNNING` should be mapped to `JobStatus.PENDING` now, because:
1.  ray status `PENDING` means the `ray job submit` has been called, i.e. the scheduler has submitted it.
2. ray status `RUNNING` means the `ray job submit` has started executing our generated ray program. Since the order of JobStatus has been changed to `PENDING` -> `SETTING_UP`, the ray status `RUNNING` need to map to the smallest status `JobStatus.PENDING`. Otherwise, a job that is pending for the resources will be set to `SETTING_UP` status in L553-L557.